app/models/concerns/request_item.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# Module for SolrDocument adding methods related to integration
+# with DUL Requests System (requests.library.duke.edu)
+
+module RequestItem
+  extend ActiveSupport::Concern
+
+  def request_item_url
+    base = DulArgonSkin.request_base_url
+    [base, 'item', stripped_id(id)].join('/')
+  end
+
+  def duke_requestable?
+    duke_owned? && physical_item?
+  end
+
+  private
+
+  def stripped_id(id)
+    id.sub('DUKE', '')
+  end
+
+  def duke_owned?
+    record_owner == 'duke'
+  end
+
+  def physical_item?
+    fetch('access_type_a').include?('At the Library')
```

Comment:
@seanaery looks like we're missing a field constant for this field in Argon. I'll add a ticket to fix that so this is fine for now. It would be better to set a default other than `nil` for fetch so that it returns an empty array in case there's no value for access_type_a: `fetch('access_type_a', []).include?('At the Library')`. I think calling `include?` on nil will cause a fatal error.